style.css

```diff
@@ -0,0 +1,238 @@
+  flex-direction: column;
+  align-items: center;
+  padding: 15px 20px;
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  background-color: #eee;
+}
+
+.logo {
+  display: flex;
+}
+
+.logo img {
+  width: 18%;
+  border-radius: 50%;
+}
+
+#nav-bar {
+  display: flex;
+  flex-grow: 1;
+  min-height: 80px;
+  align-items: center;
+}
+
+footer > ul {
+  padding: 0;
+}
+
+footer > ul li {
+  display: inline;
+}
+
+footer > ul li a {
+  color: #000;
+  text-decoration: none;
+}
+
+#nav-bar ul {
+  text-align: center;
+  padding: 0;
+}
+
+#nav-bar ul li {
+  display: block;
+  margin-bottom: 20px;
+}
+
+#nav-bar ul li a {
+  color: #000;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+/* ======= form ================= */
+
+.form {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 150px;
+}
+
+.form form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 430px;
+  width: 100%;
+}
+
+.form form input {
+  display: block;
+}
+
+.form form input:first-child {
+  margin-bottom: 20px;
+  max-width: 300px;
+  background-color: #fff;
+  padding: 5px;
+  width: 100%;
+}
+
+.form form input:last-child {
+  margin-bottom: 20px;
+  background-color: #f1c40f;
+  padding: 4px 20px;
+  font-size: 18px;
+  border: none;
+  border-radius: 4px;
+}
+
+/* ====== features ===== */
+
+#features {
+  max-width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#features .card {
+  display: flex;
+  align-items: center;
+}
+
+#features .card .icon {
+  max-width: 100px;
+  width: 100%;
+  color: #ff8c00;
+  display: none;
+}
+
+#features .card .conten {
+  flex-grow: 1;
+  text-align: center;
+}
+
+/* ======= how it works ====== */
+#how-it-works {
+  max-width: 100%;
+  margin: 50px auto;
+}
+
+/* ===== Pricing ==== */
+#pricing {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+#pricing .card {
+  margin: 0 auto;
+  margin-bottom: 15px;
+  border: 1px solid #000;
+  max-width: 340px;
+  width: 100%;
+  border-radius: 4px;
+  text-align: center;
+}
+
+#pricing .card h2 {
+  background-color: #ddd;
+  color: black;
+  padding: 15px 0;
+  width: 100%;
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0;
+  margin-bottom: 15px;
+}
+
+#pricing .card strong {
+  font-size: 24px;
+}
+
+#pricing .card button {
+  border: 0;
+  margin: 15px 0;
+  background-color: #f1c40f;
+  padding: 8px 20px;
+  border-radius: 2px;
+  cursor: pointer;
+  font-size: 24px;
+}
+
+/* ======= footer ========== */
+
+footer {
+  padding: 20px;
+  margin-top: 30px;
+  text-align: center;
+  background-color: #ddd;
+}
+
+/* ====== Media query ====== */
+
+@media (min-width: 550px) {
+  #features .card .icon {
+    display: block;
+  }
+
+  #features .card .content {
+    text-align: left;
+  }
+}
+
+@media (min-width: 600px) {
+  #header {
+    flex-direction: row;
+    left: 5%;
+    right: 5%;
+  }
```

Comment:
[OPTIONAL]  
Please can you move this part to the end, this makes your code look neat and clean